client/.env

```diff
@@ -1 +1 @@
-REACT_APP_BACKEND_URL=http://localhost:8000/api/v1/
\ No newline at end of file
+REACT_APP_BACKEND_URL=http://localhost:8000/api/v1
```

Comment:
exclude `.env` file, but retain the `.env.example` since it is the basis if another person will check your repo